### PR TITLE
test: fix workers test

### PR DESCRIPTION
**Pull request by aslushnikov**

### CODE REVIEW

File: test/worker.spec.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ module.exports.addTests = function({testRunner, expect}) {
       expect(await worker.evaluate(() => self.workerFunction())).toBe('worker function result');
 
       await page.goto(server.EMPTY_PAGE);
-      expect(page.workers()).toEqual([]);
+      expect(page.workers().length).toBe(1);
```
Comment (JoelEinbinder) [inline test/worker.spec.js]: This doesn't make sense to me. Isn't it the opposite of what the test is testing for? EMPTY_PAGE shouldn't have any workers.

Comment (aslushnikov) [reply test/worker.spec.js]: Ah, I meant 0.

File: test/worker.spec.js [hunk 2]
```diff
@@ -15,7 +15,7 @@ module.exports.addTests = function({testRunner, expect}) {
       expect(await worker.evaluate(() => self.workerFunction())).toBe('worker function result');
 
       await page.goto(server.EMPTY_PAGE);
-      expect(page.workers()).toEqual([]);
+      expect(page.workers().length).toBe(0);
```
Comment (JoelEinbinder) [inline test/worker.spec.js:18]: I don't get it. Whats the change?

Comment (aslushnikov) [reply test/worker.spec.js:18]: when this test fails, the output is cryptic since Worker object is not nicely serializable.

Comment (JoelEinbinder) [reply test/worker.spec.js:18]: Fair

**Review by JoelEinbinder — APPROVED** [review]

**Pull request merged**